packages/components/src/tools-panel/styles.ts

```diff
@@ -145,3 +145,19 @@ export const ToolsPanelItemPlaceholder = css`
 export const DropdownMenu = css`
 	min-width: 200px;
 `;
+
+export const DefaultControlsItem = css`
+	span:last-child {
+		color: var( --wp-admin-theme-color-darker-10 );
+		margin-left: ${ space( 3 ) };
```

Comment:
What do you think about making this margin-left come as a part of the `suffix` prop?

Reply:
By "come as part of the suffix prop", do you mean that the MenuItem component would apply a left margin here by default? I could see different suffixes desiring no margin or maybe something different. Could the button itself also be responsible for controlling the spacing between the inner elements?

All that said, I'm happy to make changes here.

Reply:
If I understand Lena's request correctly, she may have meant that the `margin-left` would be applied by `MenuItem` only when the `suffix` prop is defined. But I'll let her clarify tomorrow when she's back.

> I could see different suffixes desiring no margin or maybe something different. Could the button itself also be responsible for controlling the spacing between the inner elements?

Interesting! What scenarios do you have in mind?

Finally, regardless of where we use `margin-left`, we should make sure if will work well for RTL layouts (ie. potentially wrapping it in a `rtl` call).

---

Potentially, as a follow-up, we could consider refactoring `MenuItem` to use `HStack` and the `gap` prop:

- we could expose the `gap` prop as a way to let consumers tweak spacing around the suffix
- RTL layouts would automatically taken care of (because of `flexbox`)

Reply:
> What scenarios do you have in mind?

My comment was more in the general sense and not trying to assume I'll know what consumers might wish to render in a menu item. I've wrapped the `margin-left` in an `rtl` call for now (https://github.com/WordPress/gutenberg/pull/44260/commits/6ab67b83c3d19b940fcc0982cdb7cc355939b72d) to move this forward.

> Potentially, as a follow-up, we could consider refactoring MenuItem to use HStack and the gap prop:

I like the idea of reworking `MenuItem` to leverage `HStack`. It's along the lines I was thinking suggesting the button could handle the spacing.

Reply:
Yep, you both understood me correctly 🙂 It seems like all the suffix-like bits of our menu items in Gutenberg currently do have left margins, so I'm leaning towards adding a reasonable amount by default to the MenuItem suffix. (Not exactly a blocker though, if we want to get this PR merged.)

In terms of customizable whitespace, we have been trying out "utility wrapper components" as a way to handle custom padding needs cleanly without bloating the main component API. (For example in [InputControl](https://wordpress.github.io/gutenberg/?path=/docs/components-experimental-inputcontrol--with-prefix) and [Dropdown](https://wordpress.github.io/gutenberg/?path=/docs/components-dropdown--with-no-padding).) So that is also a possible approach, if the need ever arises. But currently I don't think it's likely that we'll have a no-padding use case here.

Reply:
_I'll take over from Lena as she's going to be AFK for the next week_

Let's not consider this as blocking for this, although it would be nice to work on follow-up in which:

- we move the suffix margin to `MenuItem`
- we look into using `HStack` for `MenuMargin`'s layout
- we consider exposing a `gap` prop if necessary, to tweak horizontal spacing inside `MenuMargin`

How does that sound?

Reply:
Thanks @ciampo, that sounds like a plan to me 👍 

The only thing I'd add is we could potentially convert MenuItem to TypeScript if that helps bring it in line with the rest of the components package.

Reply:
That would be great, although it can also be definitely done separately — up to you!